source/Plugins/EpicLibrary/Services/EpicAccountClient.cs

```diff
@@ -233,6 +247,11 @@ private void renewToknes(string refreshToken)
         {
             using (var httpClient = new HttpClient())
             {
+                if (url.Contains("/library/api/public/playtime/account/"))

```

Comment:
I don't think this should be placed here. Generic method like this should not be doing arbitrary modifications to passed parameters. Url modifications should be done in GetPlaytimeItems.

Reply:
I agree. I was not sure if this was a good idea for that reason but decided this initially to not have to load the tokens twice.

Reply:
Updated PR with the change